timesketch/frontend-ng/src/components/LeftPanel/ThreatIntel.vue

```diff
@@ -0,0 +1,238 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <v-row
+      no-gutters
+      style="cursor: pointer"
+      class="pa-4"
+      flat
+      @click="expanded = !expanded"
+      :class="$vuetify.theme.dark ? 'dark-hover' : 'light-hover'"
+    >
+      <span
+        ><v-icon left>mdi-shield-search</v-icon> Threat Intelligence (<small
+          ><strong>{{ intelligenceData.length }}</strong></small
+        >)</span
+      >
+      <v-spacer></v-spacer>
+
+      <v-btn
+        v-if="expanded"
+        small
+        depressed
+        color="primary"
+        outlined
+        :to="{ name: 'Intelligence', params: { sketchId: sketch.id } }"
+        @click.stop=""
+      >
+        <v-icon small left>mdi-pencil</v-icon>Manage
+      </v-btn>
+    </v-row>
+
+    <v-expand-transition>
+      <div v-show="expanded">
+        <v-divider></v-divider>
+
+        <v-tabs grow v-model="tabs">
+          <v-tab
+            >Indicators (<small>{{ intelligenceData.length }}</small
+            >)</v-tab
+          >
```

Comment:
What's up with this awkward indentation? is it the auto format? I've noticed some similar behavior, but usually editing the HTML so that it kinda makes sense (e.g. closing brackets on the same line) fixes the autoformat behavior.

Reply:
Yeah, this is the auto format. I'm also trying to fix but sometimes it slips through.